Userland/Services/WindowServer/Compositor.cpp

```diff
@@ -535,7 +536,7 @@ void Compositor::compose()
     // Paint the window stack.
     if (m_invalidated_window) {
         auto* fullscreen_window = wm.active_fullscreen_window();
-        if (fullscreen_window && fullscreen_window->is_opaque()) {
+        if (fullscreen_window && fullscreen_window->is_opaque() && !WindowSwitcher::the().is_visible()) {
```

Comment:
It feels a bit weird to make an exception for one specific component that needs to be on top of all windows - couldn't the WindowSwitcher be an overlay instead?

Reply:
I agree. There is actually a FIXME for that in WindowSwitcher.cpp@172 which states "That would require adding support for event handling to overlays".
I felt this small PR would be a start, and once I'm more familiar with the codebase, maybe I can tackle that FIXME.
Also, as WindowSwitcher is a component inside WindowManager, I thought it would be OK.

Reply:
For now, can you add a FIXME to each of these places that check `WindowSwitcher::the().is_visible()`? That way it's clearer that it's not a permanent solution, and someone can come along and clean it up once the WindowSwitcher is an overlay.

Reply:
Done!